cmds/grpc-backend/main.go

```diff
@@ -156,6 +154,8 @@ func RunGRPCServer(ctx context.Context, address string) error {
 		scopesValidators = auth.MergeOperationsAndScopesValidators(scopesValidators, scdServer.AuthScopes())
 	}
 
+	MustSupportSchema(ctx, store)
```

Comment:
why move this?

Reply:
If SCD is run at this commit, it needs to fail if the database is at a lower schema version than the schema altered in this PR.  We previously accomplished this task with 8f2687c and this PR follows that pattern.  This statement needed to be moved because [the check](https://github.com/interuss/dss/blob/c90bee80069b1376bb45034eb0da16c17ecf0572/pkg/rid/cockroach/store.go#L192) must follow the SCD bootstrapping.  Again, I would prefer to wait for #328 and successors, but if that can't be finished in time, we'll need to take the existing approach to database schema validation.